pkgs/native_assets_cli/tool/dependencies.dart

```diff
@@ -16,12 +16,36 @@ const packages = [
 
 const pathDependencies = 'path-dependencies';
 
+const publishedDependency = 'published-dependency';
+
+const commands = [
+  pathDependencies,
+  publishedDependency,
+];
+
 // Print all command-line arguments that are Dart files.
 void main(List<String> arguments) async {
-  if (arguments.length == 1 && arguments.single == pathDependencies) {
-    await switchAllToPathDependencies();
-  } else {
-    print('The only command available is "$pathDependencies".');
+  final command = arguments.firstOrNull;
+  switch (command) {
+    case pathDependencies:
+      return await switchAllToPathDependencies();
+    case publishedDependency:
+      if (arguments.length != 3) {
+        print('Usage: $publishedDependency <package-name> <new-version>');
+        return;
+      }
+      final packageName = arguments[1];
+      if (!packages.contains(packageName)) {
+        print('Wrong package name.');
+        return;
+      }
+      final newVersion = arguments[2];
```

Comment:
Consider GETting the latest version from pub if not explicitly provided.

Reply:
Yeah many ways to improve this script until there are no manual steps. Will do that as I use the script in the future.